dev/helm/README.md

```diff
@@ -137,6 +139,8 @@ By default, PostgreSQL is installed as part of the chart.
 
 To use an external PostgreSQL server, set `postgresql.enabled` to `false` and then set `postgresql.postgresqlHost` and `postgresql.postgresqlPassword`. The other options (`postgresql.postgresqlDatabase`, `postgresql.postgresqlUser` and `postgresql.postgresqlPort`) may also want changing from their default values.
 
+To use an SSL connection you can set `postgresql.ssl` to `true` and if needed the path to a Certificate of Authority can be set using `postgresql:ca` to `/path/to/ca`. Default `postgresql.ssl` value is `false`.
```

Comment:
You still have the `postgres:ca` typo, should be `postgres.ca`

Reply:
oh man seeing things, my bad